src/components/parts/Input.tsx

```diff
@@ -14,6 +15,8 @@ interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
   isPhone?: boolean;
   isPhonePrefix?: boolean;
   isBirth?: boolean;
+  isNickname?: boolean;
+  isNumber?: boolean;
```

Comment:
이런식으로 인풋 종류로 관리하는 것도 괜찮을 것 같습니당

```suggestion
  isPhonePrefix?: boolean;
    type: 'phone' | 'birth' | 'nickname' | 'number'  
```